app/shared/src/forms/datePicker/DatePicker.ios.js

```diff
@@ -0,0 +1,81 @@
+// @flow
+
+import * as React from 'react';
+import { DatePickerIOS, NativeModules } from 'react-native';
+import Translation from '@kiwicom/react-native-app-translations';
+
+import DatePickerButton from './DatePickerButton';
+import BarPopup from '../../popup/BarPopup';
+
+type Props = {
+  date: Date,
+  minDate?: Date,
+  maxDate?: Date,
+  format?: string,
+  onDateChange: (date: Date) => void,
+};
+
+type State = {|
+  isPickerOpen: boolean,
+  date: Date,
+|};
+
+export default class DatePicker extends React.Component<Props, State> {
+  static defaultProps = {
+    format: 'MMM DD',
```

Comment:
This whole component seems like it behaves completely differently from Android (for example this format). I would expect to have same interfaces and behavior just platform specific...

Reply:
I started making the ios picker, this I moved this defaultProps down to `DatePickerButton` and forgot to remove it here. 

The components needs to behave differently since the underlying code is different, `DatePickerIOS` vs `DatePickerAndroid`

They take the same props so they are used the same from the outside

Reply:
> They take the same props so they are used the same from the outside

This is what I mean by "same interfaces" and by the "same behavior" I mean "do not format the dates here and not in the Android component"... :)